### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by veronica365**
The PR allows adding 2 sections needed to build for this project:

- the toolbar (or header): Toolbar
- the headline section (right after the header):

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,17 +5,57 @@
...
 
 <body>
     <link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
     <noscript>
         <link rel="stylesheet" href="styles.css">
     </noscript>
-    <h1 class="header">Hello Microverse!</h1>
+    <div class="wrap">
+        <header class="toolbar">
+            <h4>My Logo</h4><img src="assets/img/bugger.png" alt="" />
```
Comment (didierganthier) [inline index.html]: Hey 👋🏽  great design 🚀  but let's see how we can improve it:
- [ ] The logo could be on the left and have a link (a tag).
    - This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
- [ ] The hamburger button could be on the right, and have a nav tag.
    - The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @veronica365 ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: index.html (hunk 1 above)
Comment (veronica365) [reply index.html]: Hey @didierganthier I have implemented the feedback and added this screenshot for context

screens

File: index.html [hunk 2]
```diff
@@ -5,17 +5,64 @@
...
 <body>
     <link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
     <noscript>
         <link rel="stylesheet" href="styles.css">
     </noscript>
-    <h1 class="header">Hello Microverse!</h1>
+    <div class="wrap">
+        <header class="toolbar">
+            <a href="#">
+                <h4>My Logo</h4>
```
Comment (Nyame-Wolf) [inline index.html:26]: [OPTIONAL]Great job implementing the requested change. You could remove the `h4` entirely and just have it as `<a href="#">My Logo </a>.` It makes more sense using elements when you really need them

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @veronica365  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- Great use of gitflow
- Linter checks are passing
- Follows figma template

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**